integreat_cms/cms/templates/organizations/organization_list.html

```diff
@@ -5,20 +5,48 @@
     <div class="table-header">
         <div class="flex flex-wrap justify-between">
             <h1 class="heading">
-                {% translate "Manage Organizations" %}
+                {% if is_archive %}
+                    {% translate "Archived Organizations" %}
+                {% else %}
+                    {% translate "Manage Organizations" %}
+                {% endif %}
             </h1>
-            {% if perms.cms.change_organization %}
-                <a href="{% url 'new_organization' region_slug=request.region.slug %}"
-                   class="btn">
-                    {% translate "Create organization" %}
-                </a>
-            {% endif %}
+            <div class="justify-end">
+                {% if is_archive %}
+                    <a href="{% url 'organizations' region_slug=request.region.slug %}"
+                       class="font-bold text-sm text-gray-800 flex items-center gap-1 mb-2 hover:underline">
+                        <span><i icon-name="umbrella" class="align-top h-5"></i> {% translate "Back to organizations" %}</span>
+                    </a>
+                {% else %}
+                    <a href="{% url 'archived_organizations' region_slug=request.region.slug %}"
+                       class="font-bold text-sm text-gray-800 flex items-center pb-3 hover:underline">
+                        <span>
+                            <i icon-name="archive" class="align-top h-5"></i>
+                            {% translate "Archived organizations" %}
+                            ({{ archived_count }})
+                        </span>
+                    </a>
+                {% endif %}
+            </div>
         </div>
+        {% if not is_archive %}
+            <div class="flex justify-end">
+                {% if perms.cms.change_organization %}
+                    <a href="{% url 'new_organization' region_slug=request.region.slug %}"
+                       class="btn">
+                        {% translate "Create organization" %}
+                    </a>
+                {% endif %}
+            </div>
+        {% endif %}
     </div>
     <div class="table-listing">
         <table class="w-full mt-4 rounded border border-solid border-gray-200 shadow bg-white">
```

Comment:
```suggestion
        <form id="bulk-action-form" method="post">
            {% csrf_token %}
        </form>
        <table class="w-full mt-4 rounded border border-solid border-gray-200 shadow bg-white">
```

Without this form the checkboxes won't work